packages/common/src/converter/payload-converters.ts

```diff
@@ -1,19 +1,75 @@
-import { errorMessage, UnsupportedJsonTypeError, ValueError } from '@temporalio/internal-workflow-common';
```

Comment:
All of the helper methods here should accept WrappedPayloadConverter to make sure we never use an unwrapped one IMHO.
WDYT?